### PR TITLE
[WIP] Literate console target

### PR DESCRIPTION
**Pull request by adamchester**
Work in progress, no need for reviews yet 👍

### CODE REVIEW

**Comment by adamchester** [comment]
Updated exception new lines, and started on object (map) rendering
img_0188

**Comment by adamchester** [comment]
image

**Comment by adamchester** [comment]
image

**Comment by adamchester** [comment]
- [ ] Fix exception prefix on Mono

image

File: examples/Servizz/Program.fs [hunk 1]
```diff
@@ -10,14 +10,23 @@ open System.Threading
 
 let logger = Logging.getLoggerByName "Servizz.Program"
 
+type Speed = Fast | Slow | OtherSpeed of speed:int
+type LoginLogoutEvent =
+| UserLoggedIn of user : string * roles : string list
```
Comment (haf) [inline examples/Servizz/Program.fs]: 2 space indentation

File: src/Logary/TemplateFormat.fs [hunk 2]
```diff
@@ -0,0 +1,101 @@
+﻿[<AutoOpen>]
+module TemplateFormat
```
Comment (haf) [inline src/Logary/TemplateFormat.fs]: Why not make this a single module, like so: `[<AutoOpen>] module internal Logary.LogaryCapturing`?

File: src/Logary/TemplateFormat.fs [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+  let logyarDefaultDestructurer : Destructurer =
+    Capturing.createCustomDestructurer (None) (Some Capturing.destructureFSharpTypes)
+
+  let captureField (property : Utils.FsMessageTemplates.Property) value =
+    logyarDefaultDestructurer (DestructureRequest (logyarDefaultDestructurer, value, 10, 0, property.Destr))
+    |> convertTemplatePropertyToField
+    |> fun f -> Field (f, None)
+
+type Message with
```
Comment (haf) [inline src/Logary/TemplateFormat.fs]: @mavnn Compared to *Xenogloss*, what do you think of this? I think it's a good trade-off in that it doesn't add any dependencies; but there may be easy gain to be had by adding some of your code, perhaps?

**Comment by adamchester** [comment]
Fixed exception stack trace prefix on mono:
image

**Comment by haf** [comment]
This seems ready to merge for me :) – do you agree?

**Comment by adamchester** [comment]
I have a whole bunch of tests to add yet, and potentially some bits I haven't considered fully yet. I'm OK with merge & fix later, in smaller increments, if you think that's wise.

**Comment by haf-lynx** [comment]
Especially tests are good, but I think this is good enough, since the coloured target is currently opt-in rather than default. I'm also considering giving you commit access to the repo; what would you say about that? :)

**Comment by adamchester** [comment]
Ok 👍

**Pull request merged**